packages/app/components/Today/OpenQueueCard.tsx

```diff
@@ -119,71 +127,55 @@ const OpenQueueCard = ({
 
   return (
     <PaddedCard
-      headStyle={{ background: "#F3F5F7" }}
+      headStyle={{ background: "#25426C", color: "#FFFFFF" }}
       className={"open-queue-card"}
-      title={
-        staffList.map((staffMember) => staffMember.name).join(", ") ||
-        "No Staff Checked In!"
-      }
-      extra={queue.startTime && queue.endTime && formatQueueTime(queue)}
+      title={`${queue.room} ${
+        queue.isProfessorQueue ? `(Professor queue)` : ``
+      }`}
+      extra={`${queue.queueSize} in queue`}
     >
       <QueueInfoRow>
-        <HeaderDiv>{queue.room}</HeaderDiv>
+        <HeaderDiv>
+          <QuestionNumberSpan>{queue.staffList.length}</QuestionNumberSpan>{" "}
+          staff checked in
+        </HeaderDiv>
         <RightQueueInfoRow>
-          {!queue.allowQuestions && (
-            <Tooltip title="This queue is no longer accepting questions">
-              <StopOutlined
-                style={{ color: "red", fontSize: "24px", marginRight: "8px" }}
-              />
-            </Tooltip>
-          )}
-          <QueueSizeColorDiv>
-            <QuestionNumberSpan>{queue.queueSize}</QuestionNumberSpan> in queue
-          </QueueSizeColorDiv>
+          <Space
+            direction="vertical"
+            size="middle"
+            style={{ display: "flex", alignItems: "center" }}
+          >
+            {!queue.allowQuestions && (
+              <Tooltip title="This queue is no longer accepting questions">
+                <Tag icon={<StopOutlined />} color="error">
+                  Not Accepting Questions
+                </Tag>
+              </Tooltip>
+            )}
+            <Link
+              href="/course/[cid]/queue/[qid]"
+              as={`/course/${cid}/queue/${queue.id}`}
+            >
+              <OpenQueueButton
```

Comment:
should we disable this button if it's not allowing questions/queue is disabled?

Reply:
good point, afaik TA's still can check in though even if allowing questions is disabled